scripts/lib/check_rabbitmq_queue.py

```diff
@@ -80,6 +80,28 @@ def analyze_queue_stats(
             ),
         )
 
+    if (
+        old_stats
+        and now - stats["update_time"] <= 180
+        and stats["recent_average_consume_time"] is None
+        and now - old_stats["update_time"] > 180
+        and queue_count_rabbitmqctl > 10
+    ):
```

Comment:
I'm worried that this algorithm will result in this check failing every time the digest worker starts doing things for the first time the day after a production deployment, at least until the first N events are added.  The prod deployment will result in a stats file with `recent_average_consume_time=None` being generated for the worker's new startup, and yesterday's data being moved to the `.old` file ... and then 12 hours later when N events get put into the digest queue, all of these checks will pass, at least until the digest worker processes enough events to write a new stats file.  (Though maybe that's just one event, since we'll be well past `MAX_SECONDS_BEFORE_UPDATE_STATS`?)

I guess the previous `if` block will have fired in this situation anyway, as it's the exact same race discussed there, so this is OK.  Do I model this correctly, and do we want a test for this corner case?

Reply:
@timabbott Hmm, this is a different case from the codeblock above, but also unlikely I think? We have the condition `now - stats["update_time"] <= 180` here - meaning that we would need that digest emails pile of events to land in the queue within first 3 minutes from the new prod deployment - which is unlikely, and I think in the worst case, a false alarm 3 minutes after deploying is not a significant issue?